tests/test_utils.py

```diff
@@ -1,12 +1,27 @@
 # coding=utf-8
 """Unit tests for :mod:`camayoc.utils`."""
+import os
 from unittest import mock
 
 import pytest
 
 from camayoc import exceptions, utils
 
 
+def test_run_scans():
+    """Test the RUN_SCANS variable is correctly detected.
+
+    This environment variable allows users to disable tests that run
+    scans temporarily.
+    """
+    os.environ['RUN_SCANS'] = 'FALSE'
+    assert utils.run_scans() == False # noqa
```

Comment:
I am wondering why the noqa?

Reply:
I was interested in asserting that it was a boolean `False` ( not just "falsey" like `None`)